README.md

```diff
@@ -12,6 +12,9 @@ The Cloud Foundation toolkit (henceforth, CFT) includes the following parts:
   single or multiple CFT-compliant config files - see:
   - The CFT source Python files (the `src/` directory)
   - The [CFT DM User Guide](dm/docs/userguide.md)
+- A growing set of [Config
+  Connector](https://cloud.google.com/config-connector/docs/overview)
+  YAML deployments: - see [the solutions directory](config-connector/solutions/)
```

Comment:
I think because deployment is a specific concept in K8s, the word is being overloaded a bit, you might want to pick a different word. I actually think what Alex suggested before, "examples", is a pretty good summary of what they are.

Reply:
sample configurations or solutions